cmd/livekit-cli/loadtest.go

```diff
@@ -80,56 +80,54 @@ var LoadTestCommands = []*cli.Command{
 	},
 }
 
-func loadTest(c *cli.Context) error {
-	pc, err := loadProjectDetails(c)
+func loadTest(cCtx *cli.Context) error {
```

Comment:
Sticking to the [examples](https://cli.urfave.org/v2/examples/full-api-example/) from `urfave/cli`.